plugins/decklink/decklink-device-instance.cpp

```diff
@@ -175,11 +193,17 @@ bool DeckLinkDeviceInstance::StartCapture(DeckLinkDeviceMode *mode_)
 		if (audioResult != S_OK)
 			LOG(LOG_WARNING, "Failed to enable audio input; continuing...");
 
-		if (!ISSTEREO(channelFormat)) {
-			const audio_repack_mode_t repack_mode = ConvertRepackFormat(channelFormat);
+		if ((channelFormat != SPEAKERS_UNKNOWN) &&
+			(channelFormat != SPEAKERS_MONO) &&
+			(channelFormat != SPEAKERS_STEREO) && 
+			(channelFormat != SPEAKERS_2POINT1)	&& (maxdevicechannel >= 8)) {
+
+			const audio_repack_mode_t repack_mode = ConvertRepackFormat
+			(channelFormat);
 			audioRepacker = new AudioRepacker(repack_mode);
-		}
-	}
+			}
```

Comment:
I'm pretty sure that this closing curly brace and the one below it are misaligned.